src/main/java/kostka/moviecatalog/controller/MovieCatalogController.java

```diff
@@ -16,29 +16,29 @@
 
 @RestController
 public class MovieCatalogController {
-    private MovieService movieService;
+    private MovieServiceImpl movieServiceImpl;
     static final Logger LOGGER = LogManager.getLogger("CONSOLE_JSON_APPENDER");
 
     @Autowired
-    public MovieCatalogController(final MovieService movieService) {
```

Comment:
Tady ti nešel injektovat ten interface, jo?

Mělo by to jít nějak takhle MoviceService<Movie> movieService, když už to máš s tou generikou, si myslím. Ale možná se pletu.

Reply:
Já jsem ten interface přidával až dneska, tajze jsem to zapomněl upravit nejspíš...
Zase budu muset vymyslet, jak se volá správná implementace te service kterou budu používat

Reply:
Jasan :) Jinak dobrý, líp bych to asi neudělal.

Reply:
upraveno